ownership-chain/runtime/src/precompiles/mod.rs

```diff
@@ -61,18 +49,12 @@ where
 			// Non-Frontier specific nor Ethereum precompiles :
 			// a if a == hash(1024) => Some(Sha3FIPS256::execute(handle)),
 			a if a == hash(1025) => Some(ECRecoverPublicKey::execute(handle)),
-			a if a == hash(1026) => Some(LivingAssetsPrecompile::execute(handle)),
```

Comment:
should be removed from `pub fn used_addresses()` as well

Reply:
done